src/app/system/global-configurations/edit-configuration/edit-configuration.component.ts

```diff
@@ -0,0 +1,71 @@
+/** Angular Imports */
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { ActivatedRoute, Router } from '@angular/router';
+
+/** Custom Services */
+import { SystemService } from '../../system.service';
+
+@Component({
+  selector: 'mifosx-edit-configuration',
+  templateUrl: './edit-configuration.component.html',
+  styleUrls: ['./edit-configuration.component.scss']
+})
+export class EditConfigurationComponent implements OnInit {
+
+  /** Edit Configuration form. */
+  configurationForm: FormGroup;
+  /** configuration data */
+  configuration: any;
+
+  /**
+   * @param {FormBuilder} formBuilder Form Builder.
+   * @param {SystemService} systemService System Service.
+   * @param {ActivatedRoute} route Activated Route.
+   * @param {Router} router Router for navigation.
+   */
+  constructor(private formBuilder: FormBuilder,
+              private systemService: SystemService,
+              private route: ActivatedRoute,
+              private router: Router) {
+    this.route.data.subscribe((data: { configuration: any }) => {
+      this.configuration = data.configuration;
+    });
+  }
+
+  /**
+   * Creates the edit configuration form.
+   */
+  ngOnInit() {
+    this.createConfigurationForm();
+    this.setConfigurationForm();
```

Comment:
Set the value while building the form. Do not use setters, they are unnecessary.